packages/block-library/src/columns/editor.scss

```diff
@@ -35,7 +35,7 @@
 		// Responsiveness: Allow wrapping on mobile.
 		flex-wrap: wrap;
 
-		@include break-small() {
+		@include break-medium() {
```

Comment:
We discussed this a little bit already, but what is the context for changing this to medium? 

To my understanding the ideal situation is this:

- Mobile (small) breakpoint has 1 column
- Medium breakpoint has 2 columns
- > Medium has user set amount of columns.

To me this says that Mobile should _wrap_, but anything above mobile should _not_ wrap.

Reply:
The reason to change this to medium is so that the small-to-medium breakpoint (600px - 782px) will wrap.

It is supposed to wrap _after_ the EVEN columns, so that we get 2 columns, right?

```
Mobile:
1 (wrap)
2 (wrap)
3 (wrap)
4 (wrap)
5

Small to Medium
1, 2 (wrap)
3, 4 (wrap)
5

Medium and up
1, 2, 3, 4, 5
```
If we set 'nowrap' at the Small breakpoint (600px), then we eliminate the small-to-medium breakpoint.

I'm not opposed to removing the small-to-medium breakpoint entirely; I just didn't think that was the original intent.

Reply:
That's a solid point. Thank you for clarifying, and for the patience.